### PR TITLE
feat: `/cli` landing page for `pizza-cli`

### LINKED ISSUE #346: Feature: OpenSauced CLI landing page
### Suggested solution

In an effort to re-vitalize the CLI, we should have a `opensauced.pizza/cli` landing page similar to other install sites. For example, here's GitHub's CLI landing page:

Screenshot 2024-08-27 at 3 05 26 PM

And here's Homebrew's install landing page:

Screenshot 2024-08-27 at 3 06 07 PM

---

Requirements:

- Include simple brew installation instructions: `brew install open-sauced/tap/pizza` and "download" link to latest GitHub release (like the GitHub landing page does)
- Some copy on what the CLI is and or quickstart on using it.

Bonus:
- A screen-capture of the CLI in use

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Implements a new `/cli` page for the [pizza-cli](https://github.com/open-sauced/pizza) tool.

## Related Tickets & Documents

Closes #346 

## Mobile & Desktop Screenshots/Recordings

image

## Tier (staff will fill in)

- [x] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [ ] Tier 4

## [optional] What gif best describes this PR or how it makes you feel?
computer-missing-gif

### CODE REVIEW

**Comment by BekahHW** [comment]
I wonder if we should change where the CLI is linked to in the footer. Right now, they both link to the landing page, so it seems redundant. The "more sauce" section takes you to separate domains that have  either been redirected as subdirectories or are subdomains. My suggestion would be to have that take you to github.com/open-sauced/pizza-cli.

image

File: pages/cli/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <Heading>
+            Generate developer insights $yellow-to-orangefrom the command line
+          </Heading>
+
+          <Typography variant='subheading'>
+            Try the <code className='px-2'>pizza</code> CLI and access OpenSauced features right from your terminal. Autogenerate your CODEOWNERS and contributor insights in seconds.
+          </Typography>
+
+          <div className='flex gap-4 items-center'>
+            <code className='p-4 bg-neutral-800 rounded-xl'>
```
Comment (nickytonline) [inline pages/cli/index.tsx:44]: dx improvement (non-blocking): Consider a copy command button.

**Review by nickytonline — APPROVED** [review]
Looks great @zeucapua!

**Review by brandonroberts — APPROVED** [review]

**Comment by BekahHW** [comment]
Here's the info for the example: 

```
❯ pizza

A command line utility for insights, metrics, and all things OpenSauced

Usage:
  pizza <command> <subcommand> [flags]

Available Commands:
  generate    Generate configurations and codeowner files
  bake        Use a pizza-oven to source git commits into OpenSauced
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  login       Log into the CLI application via GitHub
  repo-query  Ask questions about a GitHub repository

Flags:
  -h, --help   help for pizza

Use "pizza [command] --help" for more information about a command.
```

File: components/sections/Footer.tsx [hunk 2]
```diff
@@ -42,6 +42,10 @@ const footerContext = {
       url: 'https://opensauced.pizza/learn',
       label: 'opensauced.pizza/learn',
     },
+    {
+      url: 'https://opensauced.pizza/cli',
+      label: 'opensauced.pizza/cli',
+    },
```
Comment (bdougie) [inline components/sections/Footer.tsx:48]:
```suggestion
```
per @BekahHW's comment this is not necessary in the footer. More sauce is all external links.

**Review by bdougie — CHANGES_REQUESTED** [review]

**Review by bdougie — APPROVED** [review]

**Comment by bdougie** [comment]
Just saw mobile breaks because the homebrew command is longer. image

File: pages/cli/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+              brew install open-sauced/tap/pizza 
+            </code>
+            <p>or</p>
+            <Button href='https://github.com/open-sauced/pizza-cli/releases'>Download for Mac</Button>
+          </div>
+
+          <a href="https://github.com/open-sauced/pizza-cli#readme" className='hover:underline'>
```
Comment (bdougie) [inline pages/cli/index.tsx:51]:
```suggestion
          <a href="https://github.com/open-sauced/pizza-cli?tab=readme-ov-file#-install" className='hover:underline'>
```
This link is more ideal

**Pull request merged**